casanovo/casanovo.py

```
@@ -621,83 +759,90 @@ def _get_model_weights(cache_dir: Path) -> Path:
+            canonical_id = _resolve_selector(
+                selector, list({mid for mid, _, _ in local})
             )
-            if match > version_match[2]:
-                version_match = os.path.join(cache_dir, filename), None, match
-    # Provide the cached model weights if found.
-    if version_match[2] > 0:
-        logger.info(
-            "Model weights file %s retrieved from local cache",
-            version_match[0],
+        except ValueError:
+            pass
```

Ruff (0.15.14)

[warning] Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(ValueError): ...`

(SIM105)